ex2_hal/sband/equipment_handler/include/sTransmitter.h

```diff
@@ -91,10 +91,11 @@
 #define S_PAPWR_30DBM 30
 
 // Frequency maximums/minimums
-#define S_FREQ_COMMERCIAL_MAX 2300.0f
-#define S_FREQ_COMMERCIAL_MIN 2200.0f
-#define S_FREQ_AMATEUR_MAX 2450.0f
-#define S_FREQ_AMATEUR_MIN 2400.0f
+#define S_FREQ_COMMERCIAL_MAX 2300000000 // Hz
```

Comment:
This looks like 2.3 Giga Hertz - just confirming that's what you meant it to be :-)

Reply:
Yes it is!